packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.test.tsx

```diff
@@ -152,4 +146,20 @@ describe('ChoiceGroup', () => {
     expect(extraAttributeGetter(0)).toEqual('auto1');
     expect(extraAttributeGetter(1)).toBeNull();
   });
+
+  it('has role="application" by default on the containing element', () => {
```

Comment:
The comment that is in the code providing context should be here in the unit test as well.